packages/mobile/src/identity/verification.ts

```diff
@@ -138,48 +138,52 @@ export function* doVerificationFlow() {
 
     CeloAnalytics.track(CustomEventNames.verification_get_status)
 
-    if (status.isVerified) {
-      yield put(setVerificationStatus(VerificationStatus.Done))
-      yield put(setNumberVerified(true))
-      return true
-    }
-
-    // Mark codes completed in previous attempts
-    yield put(completeAttestationCode(NUM_ATTESTATIONS_REQUIRED - status.numAttestationsRemaining))
-
-    yield put(setVerificationStatus(VerificationStatus.RequestingAttestations))
-    const attestations: ActionableAttestation[] = yield call(
-      requestAndRetrieveAttestations,
-      attestationsWrapper,
-      phoneHash,
-      account,
-      status.numAttestationsRemaining
-    )
+    if (!status.isVerified) {
```

Comment:
This just removes duplication of:
```
yield put(setVerificationStatus(VerificationStatus.Done))
yield put(setNumberVerified(true))
return true
```

Reply:
I preferred it as it was cause now this large code block gets further indented, but it's nbd either way